vscode/src/client.ts

```diff
@@ -224,6 +229,25 @@ class ClientErrorHandler implements ErrorHandler {
   }
 }
 
+class ExperimentalCapabilities implements StaticFeature {
```

Comment:
This might seem complicated, but all we need is adding the extra experimental capabilities in `fillClientCapabilities`. We don't use anything else.